src/main.py

```diff
@@ -20,20 +26,65 @@ def on_down_button_pressed():
     
 
 
-def on_button_released():
+def on_lift_button_released():
     lift.stop()
 
 
+
+# controller event handlers
 controller.buttonR1.pressed(on_up_button_pressed)
-controller.buttonR1.released(on_button_released)
+controller.buttonR1.released(on_lift_button_released)
 controller.buttonR2.pressed(on_down_button_pressed)
-controller.buttonR2.released(on_button_released)
+controller.buttonR2.released(on_lift_button_released)
+
+
+####### CLAW CODE ↓ ########
+
+claw = Motor(Ports.PORT12)

```

Comment:
Do you think it would be easier to move the declarations of the motors to the top?

Reply:
I actually think that it makes more sense to keep them with the rest of the code for that part. Keeping everything related to the claw with the claw I think makes it easier to keep stuff organized. But it is up to you.